application/src/main/resources/ecc.yml

```diff
@@ -53,10 +53,17 @@ connection:
     ## Allow routing requests directly to a remote datacenter.
     ## This allows locks for other datacenters to be taken in that datacenter instead of via the local datacenter.
     ## If clients are prevented from connecting directly to Cassandra nodes in other sites this is not possible.
-    ## If remote routing is disabled its not possible to use LOCAL_SERIAL consistency for the locking,
+    ## If remote routing is disabled and consistency serial uses the DEFAULT mode its not possible to use LOCAL_SERIAL consistency for the locking,
     ## instead SERIAL consistency will be used for those request.
     ##
     remoteRouting: true
+    ##
```

Comment:
Try to follow the same format as above. Something like:
Allow to override consistency level for LWT (lightweight transactions).
Possible values are:
  "DEFAULT" - Use consistency level based on remoteRouting.
  "SERIAL" - Use SERIAL consistency for LWT regardless of remoteRouting.
  "LOCAL_SERIAL" - Use LOCAL_SERIAL consistency for LWT regardless of remoteRouting.

Reply:
Done